scripts/create-a-m-o/functions.js

```diff
@@ -145,6 +145,10 @@ const createFiles = (store, a, m, o, done) => () => {
     ### onClick
 
     The function-valued attribute \`onClick\` can be used as a callback prop for React and other frameworks.
+    
+    ### Migration Notes (v1.0 -> v2.0)
+    
+    You don't have to pay attention to anything for upgrading to version 2.
```

Comment:
also here, better no version added. somethink like: `You don't have to pay attention to anything for upgrading to newer version.`